nautobot/docs/rest-api/filtering.md

```diff
@@ -85,3 +85,32 @@ String based (char) fields (Name, Address, etc) support these lookup expressions
 
 Certain other fields, namely foreign key relationships support just the negation
 expression: `n`.
+
+
+### Network and Host Fields
+
+There are [Custom Lookups](https://docs.djangoproject.com/en/3.2/howto/custom-lookups/) built for the `VarbinaryIPField` field types. While
+the `VarbinaryIPField` is applied to fields for network, host, and broadcast, the below filters only apply to network and host. The design
+makes an assumption that there is in fact a broadcast (of type `VarbinaryIPField`) and prefix_length (of type `Integer`) within the same
+model. This assumption is used to understand the relevant scope of the network in question and is important to note when extending the
+Nautobot core or plugin data model.
+
+- `**` `exact` - An exact match of an IP or network address, e.g. `host__exact="10.0.0.1"`
+- `**` `iexact` - An exact match of an IP or network address, e.g. `host__exact="10.0.0.1"`
+- `**` `startswith` - Determine if IP or network starts with the value provided, e.g. `host__startswith="10.0.0."`
+- `**` `istartswith` - Determine if IP or network starts with the value provided, e.g. `host__startswith="10.0.0."`
+- `**` `endswith` - Determine if IP or network ends with the value provided, e.g. `host__endswith="0.1"`
+- `**` `iendswith` - Determine if IP or network ends with the value provided, e.g. `host__endswith="0.1"`
+- `**` `regex` - Determine if IP or network matches the pattern provided, e.g.` host__regex=r"10\.(.*)\.1`
+- `**` `iregex` - Determine if IP or network matches the pattern provided, e.g.` host__regex=r"10\.(.*)\.1`
+- `net_contained` - Given a network, determine which networks are contained within the provided e.g. `network__net_contained="192.0.0.0/8"` would include 192.168.0.0/24 in the result
+- `net_contained_or_equal` - Given a network, determine which networks are contained or is within the provided e.g. `network__net_contained_or_equal="192.0.0.0/8"` would include 192.168.0.0/24 and 192.0.0.0/8 in the result
+- `net_contains` - Given a network, determine which networks contain the provided network e.g. `network__net_contains="192.168.0.0/16"` would include 192.0.0.0/8 in the result
+- `net_contains_or_equals` - Given a network, determine which networks contain or is the provided network e.g. `network__net_contains="192.168.0.0/16"` would include 192.0.0.0/8 and 192.168.0.0/16 in the result
+- `net_equals` - Given a network, determine which which networks are an exact match. e.g. `network__net_equals="192.168.0.0/16"`
+- `net_host` - Given an IP, determine which parent networks there are, e.g. `host__net_host="10.0.0.1"`
```

Comment:
My understanding is net_in is the only lookup that accepts a list.

Reply:
Sorry, my comment was meant to indicate that I like how the earlier items in this list give both an example of usage as well as the results that it would return - these three are missing the example results, and I'd like to see that added.

Reply:
I think I have addressed this one, let me know if I hit the mark

Reply:
There is a suggestion below where I will follow up on